tests/conftest.py

```diff
@@ -52,7 +52,7 @@ async def random_name():
     docker = Docker()
     images = await docker.images.list()
     for img in images:
-        if img["RepoTags"] is None:
+        if img["RepoTags"] in [None, []]:
```

Comment:
I'd suggest simplifying this as:
```suggestion
        if img["RepoTags"]:
```

Reply:
We want the false-y case, so I changed this to `if not img["RepoTags"]:`.

Reply:
Oh, that's correct. Thanks!